src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -323,7 +355,7 @@ class ActionCreatorEditor extends React.Component<Props, ComponentState> {
                 const action = nextProps.action
 
                 const payloadOptions = this.props.entities.map(convertEntityToOption)
-                const negativeEntityTags = convertEntityIdsToTags(action.negativeEntities, nextProps.entities)
+                let negativeEntityTags = convertEntityIdsToTags(action.negativeEntities, nextProps.entities)
```

Comment:
Why is this changed to `let`?  

Everything should be `const` unless you intend to reassign it later do to modification.

When I see `let` I think "ok this variable is required to be mutated below" then I have to remember it for the rest remainder of reading.  I see the `push` but not a re-assignment.  If that was only case, you can still push into a const and maintain assurance that no one will change the variable entirely like `negativeEntityTags = []`

Reply:
Somehow your comment got put in different spot so copying it up here for context:

> Needed to change it to let as it is now changed in the code later on in the function call.

Hmm, can you link to the line? I can't find it, only see the `push`

Reply:
Yes, the push.  Can't push onto a const, correct?

Reply:
You can.  `const` only means single assignment.  You can still mutate the nested properties of objects or array values.

Reply:
ah... thx